packages/central-server/src/scheduledTasks/TaskOverdueChecker.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import { sendEmail } from '@tupaia/server-utils';
+import { format } from 'date-fns';
+import winston from 'winston';
+import { ScheduledTask } from './ScheduledTask';
+
+export class TaskOverdueChecker extends ScheduledTask {
+  getSchedule() {
+    return '0 * * * *'; // every hour
+  }
+
+  getName() {
```

Comment:
Thoughts on making it possible to just set `this.name = XXX` and `this.schedule = XXX`, similar to how we set things in `central-server` CRUD handlers?

Reply:
yes good point. I liked the functions so that I could have the nice errors but I found another easy way of handling that.